build/device.mk

```diff
@@ -381,5 +381,5 @@ $(call inherit-product, vendor/epam/aosp-hmi/epam-automotive-launcher.mk)
 $(call inherit-product, vendor/epam/ces-navigation/epam-navigation.mk)
 #$(call inherit-product, vendor/epam/EpamCluster/epam-cluster-product.mk)
 $(call inherit-product, vendor/epam/EpamMusic/epam-audio.mk)
-$(call inherit-product, vendor/epam/EpamHNAV/epam-hnav.mk)
+#$(call inherit-product, vendor/epam/EpamHNAV/epam-hnav.mk)
```

Comment:
Hi @DKlimenko, you've uncommented this line here: https://github.com/xen-troops/android_device_xenvm/pull/92/commits/9afe714e2d776e7b60eb962f5a65337eb31b122c

Why do you need this change as a separate commit? I think it's not valid. Please, remove your "experiments", and leave only a set of final valuable changes.

Reply:
Hi. Squashed commits. Not actual any more